commons-component-product/src/test/java/org/exoplatform/commons/info/test/TestProductInformations.java

```diff
@@ -91,6 +93,8 @@ public void testUnlockInformation() {
     assertEquals(productInformations.getDuration(), "0");
     assertEquals(productInformations.getProductCode(), "0");
     assertEquals(productInformations.getProductKey(), "0");
+    assertEquals(productInformations.getProductName(), null);
+    assertEquals(productInformations.getProductLink(), null);
```

Comment:
Please make sure to add properties in `product_new.properties` and then make assertions that those values where considered instead of the old ones.

Reply:
Seems not considered, please try adding entries in file `product_new.properties` and then test whether it was imported on init like other Test Cases for other properties.